lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -3591,3 +3591,10 @@ extension DomFinalizationRegistryExtension on DomFinalizationRegistry {
 /// Whether the current browser supports `FinalizationRegistry`.
 bool browserSupportsFinalizationRegistry =
     _finalizationRegistryConstructor != null;
+
+@JS()
+@staticInterop
+extension JSArrayExtension on JSArray {
```

Comment:
Move this closer to the definition of JSArray maybe?

Reply:
`JSArray` is defined in `dart:js_interop` :)